### PR TITLE
Return Order Service Information at Root

### PR DESCRIPTION
**Pull request by SteambreadC**

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -19,7 +19,11 @@
 def index():
     """ Root URL response """
     return (
-        "Reminder: return some useful information in json format about the service here",
+        jsonify(
+            name="Orders system REST API Service",
+            version="1.0",
```
Comment (MahamArif) [inline service/routes.py:24]: Please update the test to verify the name and version returned by the service.

**Review by MahamArif — CHANGES_REQUESTED** [review]

File: service/routes.py (hunk 1 above)
Comment (SteambreadC) [reply service/routes.py:24]: Is this really necessary? We may change the name and version a lot

Comment (MahamArif) [reply service/routes.py:24]: I see, yeah it was just an improvement. LGTM!

**Review by MahamArif — APPROVED** [review]

File: service/routes.py [hunk 2]
```diff
@@ -19,7 +19,11 @@
 def index():
     """ Root URL response """
     return (
-        "Reminder: return some useful information in json format about the service here",
+        jsonify(
+            name="Orders system REST API Service",
```
Comment (MahamArif) [inline service/routes.py:23]: Can you update this name to "Order REST API Service"?

**Review by MahamArif — CHANGES_REQUESTED** [review]

**Review by MahamArif — APPROVED** [review]

**Pull request merged**